modules/library/data/src/test/java/org/geotools/data/memory/MemoryDataStoreTest.java

```diff
@@ -911,11 +907,11 @@ public void testGetFeatureSourceRoad() throws IOException {
         assertEquals(rd12Bounds, some.getBounds());
         assertEquals(some.getSchema(), road.getSchema());
 
-        DefaultQuery query = new DefaultQuery("road", rd12Filter, new String[]{"name",});
+        DefaultQuery query = new DefaultQuery("road", rd12Filter, new String[]{"name", "geom"});
 
```

Comment:
Perhaps it was a manditory field last time. In anycase the test as you have here makes sense.